app/middleware/vendor.js

```diff
@@ -0,0 +1,27 @@
+const { dirname, join } = require('path')
+const express = require('express')
+
+const router = express.Router()
+
+// Locate packages
+const pkgHTML5Shiv = dirname(require.resolve('html5shiv/package.json'))
```

Comment:
`require.resolve` make things pretty neat! 🙌🏻 Is is worth a little `packagePath(packageName)` helper?

Reply:
Done, just for you